cypress/integration/click-to-enter-spec.js

```diff
@@ -0,0 +1,32 @@
+describe("Click To Enter component", () => {
+
+  beforeEach(() => {
+    cy.visit("http://localhost:8080")
+  })
+
+  it("Should contain text and an input in the header", () => {
+    cy
+      .get("header").get("h1").contains("VUENIVERSE").should("be.visible")
+      .get("header").get("input").should("be.visible")
+      .get("header").get("p").contains("Expand your Vue.").should("be.visible")
+  })
+
+  it("Should contain a blast-off button and additional text", () => {
+    cy
+      .get("main").get("button").contains("🤠 CLICK HERE TO BLAST OFF 🤠").should("be.visible")
+      .get("main").get("p").contains("(WARNING FLASHING LIGHTS!!)").should("be.visible")
+  })
+
+  it("Should contain a footer with component links", () => {
+    cy
+      .get("footer").get("a:first").contains("About").should("be.visible")
+      .get("footer").get("a:last").contains("Home").should("be.visible")
```

Comment:
These are going to change soon so lets keep that in mind for later. The `home` link is going to be in the header